_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -106,7 +107,7 @@ These two HTML `a` elements have the same [accessible name][] and [context][prog
 
 #### Passed Example 3
 
-These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context], and resolve to identical resources.
+These two HTML `a` elements have the same non-empty [accessible name][], empty [accessible description][] and the same [context][programmatically determined link context], and resolve to identical resources.
```

Comment:
(Minor - changing comma usage and the word "and").
I suggest that you replace this:
These two HTML `a` elements have the same non-empty [accessible name][], empty [accessible description][] and the same [context][programmatically determined link context], and resolve to identical resources.
... with this:
These two HTML `a` elements have the same non-empty [accessible name][], empty [accessible description][], the same [context][programmatically determined link context], and resolve to identical resources.